src/machine/board_microbit.go

```diff
@@ -1,15 +1,20 @@
-// +build nrf,microbit
+// +build nrf51,microbit
 
 package machine
 
+import (
+	"device/nrf"
+	"errors"
+)
+
 // The micro:bit does not have a 32kHz crystal on board.
 const HasLowFrequencyCrystal = false
 
 // Buttons on the micro:bit (A and B)
 const (
-	BUTTON  = BUTTON1
-	BUTTON1 = 5
-	BUTTON2 = 11
+	BUTTON  = BUTTONA
+	BUTTONA = 17
+	BUTTONB = 26
```

Comment:
Weird. I checked the pinout for the buttons and they were supposed to be on 5 and 11. Not sure what's wrong with the docs or why they have to be different numbers.

Reply:
The numbering of the pins on the connector is not the same as the pins on the nrf51. That is why the numbers were off. You can find the mappings here:
https://github.com/lancaster-university/microbit-dal/blob/master/inc/drivers/MicroBitPin.h#L41